src/Features/TerrainHelper.cpp

```diff
@@ -0,0 +1,134 @@
+#include "TerrainHelper.h"
+
+#include "ShaderCache.h"
+#include "State.h"
+
+void TerrainHelper::DataLoaded()
+{
+	// Get the default landscape texture set for terrain helper
+	const auto defaultLandTextureSet = RE::TESForm::LookupByEditorID<RE::BGSTextureSet>("LandscapeDefault");
+	if (defaultLandTextureSet != nullptr) {
+		spdlog::info("LandscapeDefault EDID texture set found");
```

Comment:
This is not how we log in CS. Conform to what we use. https://github.com/search?q=repo%3Adoodlum%2Fskyrim-community-shaders+logger%3A%3Ainfo&type=code

Reply:
Updated